lib/suspenders/generators/stylesheet_base_generator.rb

```diff
@@ -0,0 +1,23 @@
+require "rails/generators"
+
+module Suspenders
+  class StylesheetBaseGenerator < Rails::Generators::Base
+    source_root File.expand_path(
+      File.join("..", "..", "..", "templates"),
+      File.dirname(__FILE__))
+
+    def add_stylesheet_gems
+      gem "bourbon", "5.0.0.beta.5"
+      gem "neat", "~> 1.7.0"
+      gem "refills", group: [:development, :test]
+    end
+
+    def add_css_config
+      copy_file(
+        "application.scss",
+        "app/assets/stylesheets/application.scss",
+        force: true,
+      )
```

Comment:
> How about: copy our file over and issue a warning message?

You mean appending to their css, and changing their format? Or copying it too, as we do now?

I'm inclined to changing their file, but I'm not sure how-meaningful issues will arise from this.